tests/php/VersionedGridFieldItemRequestTest/TestExtension.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace SilverStripe\Versioned\Tests\VersionedGridFieldItemRequestTest;
+
+use SilverStripe\Core\Extension;
+use SilverStripe\Dev\TestOnly;
+use SilverStripe\Forms\FieldList;
+use SilverStripe\Forms\FormAction;
+
+/**
+ * Test extension that adds actions via updateFormActions
+ */
+class TestExtension extends Extension implements TestOnly
+{
+    /**
+     * @var bool Flag to control whether to add actions
+     */
+    public static $add_test_action = false;
```

Comment:
```suggestion
     * Flag to control whether to add actions
     */
    public static bool $add_test_action = false;
```
We prefer strict types rather than PHPDoc type hints for new API. Doesn't really matter in this case but since there's likely other changes to make (i.e. removing the `return` statements if they don't do anything) this may as well get changed at the same time.